Kitodo/src/main/java/org/kitodo/services/data/TaskService.java

```diff
@@ -95,6 +97,74 @@ public static TaskService getInstance() {
         return instance;
     }
 
+    @Override
+    public List<TaskDTO> findAll(String sort, Integer offset, Integer size, Map filters) throws DataException {
+
+        LoginForm login = (LoginForm) Helper.getManagedBeanValue("#{LoginForm}");
+        if (login == null) {
+            return new ArrayList<>();
+        }
+
+        // TODO: find other way than retrieving the form bean to access "hideCorrectionTasks" and "showAutomaticTasks"
+        // e.g. which tasks should be returned!
+        BoolQueryBuilder subquery = new BoolQueryBuilder();
```

Comment:
The same query is written two times. It could be build in private method and called here by some get...Query method